extensions/wikia/Discussions/maintenance/SiteStatus.class.php

```diff
@@ -4,8 +4,6 @@
 
 
 class SiteStatus {
-	const PROD_SERVICE_URL = 'https://services.wikia.com';
-	const DEV_SERVICE_URL = 'https://services.wikia-dev.com';
```

Comment:
Looks like these removed consts are still being referred to in [getDiscussionUrl](https://github.com/Wikia/app/blob/43a4a2b887946891729f998973b4f0b83c712d17/extensions/wikia/Discussions/maintenance/SiteStatus.class.php#L360-L364).